profiler/src/ProfilerEngine/Datadog.Profiler.Native/CollectorBase.h

```diff
@@ -63,8 +63,7 @@ class CollectorBase
         _pFrameStore{pFrameStore},
         _pAppDomainStore{pAppDomainStore},
         _pRuntimeIdStore{pRuntimeIdStore},
-        _pThreadsCpuManager{pThreadsCpuManager},
-        _isTimestampsAsLabelEnabled{pConfiguration->IsTimestampsAsLabelEnabled()}
```

Comment:
The IConfiguration* parameter should be removed from the constructor signature

Reply:
done